src/commands/dev/mod.rs

```diff
@@ -42,6 +42,7 @@ pub fn dev(
     ip: Option<&str>,
     verbose: bool,
 ) -> Result<(), failure::Error> {
+    boxen::dev_alpha_warning();
```

Comment:
nit: boxen doesn't tell me anything; suggest name "flashing_neon_sign"? why we have it vs what it is, i guess.

Reply:
yeah i just used boxen because of the npm library named boxen

Reply:
sure. nice update 👍